src/main/java/de/tum/in/www1/artemis/web/rest/AchievementResource.java

```diff
@@ -47,4 +65,52 @@ public AchievementResource(AchievementService achievementService, UserService us
         return ResponseEntity.ok(achievements);
     }
 
+    /**
+     * PUT /achievements : Updates an existing achievement.
+     *
+     * @param achievement the achievement to update
+     * @return the ResponseEntity with status 200 (OK) and with body the updated achievement, or with status 500
+     *         (Internal Server Error) if the achievement couldn't be updated
+     * @throws URISyntaxException if the Location URI syntax is incorrect
+     */
+    @PutMapping("/achievements")
```

Comment:
I think you would also need here the `id` of the achievement like you have it for the delete call